Hometask8/src/main/java/homework21/CheckoutStepOnePage.java

```diff
@@ -0,0 +1,14 @@
+package homework21;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class CheckoutStepOnePage {
+    static String[] selectors = {"#first-name", "#last-name", "#postal-code"};
+    public static void fillCheckoutInformation(String[] data) {
+        for(int i = 0; i < data.length; i++) {
+            $(selectors[i]).setValue(data[i]);
+        }
+
+        $("#continue").click();
+    }
```

Comment:
method should be reworked as
1) strict dependency on passed data order
2) no ability to set values separately (may be useful for negative cases)